### DISCUSSION TITLE
SPARROW x ORD: HOW TO GET INSCRIPTIONS INTO ORD FROM SPARROW. IS IT A POSSIBILITY NOW?

### DISCUSSION THREAD

**Discussion by joeatang** [ordinals/ord#1602] [Q&A]
Macbook user asking: What is the safest way to get Inscriptions into Ord wallet? Is this even a possibility now or something that will be created in the future? If it is a possibility, what is the step by step guide to make this happen. Thank you bitcoin devs for all your hard work!

**Comment by casey** [comment]
Right now this would require manual coin selection and a good understanding of ordinal theory. I wouldn't recommend it if you're not 100% sure of what you're doing, but Sparrow supports coin selection, so it can be done.
